ignite/handlers/checkpoint.py

```diff
@@ -278,15 +285,11 @@ def __call__(self, engine: Engine) -> None:
             )
 
             if self._score_name is not None:
-                if len(suffix) > 0:
-                    suffix += "_"
-                suffix = "{}{}={}".format(suffix, self._score_name, priority_str)
-            elif self._score_function is not None:
-                if len(suffix) > 0:
-                    suffix += "_"
-                suffix = "{}{}".format(suffix, priority_str)
-            elif len(suffix) == 0:
-                suffix = "{}".format(priority_str)
+                filename_pattern_dict["score_name"] = "{}".format(self._score_name)
+                filename_pattern_dict["score"] = "{}".format(priority_str)
+
+            elif self._score_function is not None or all(suffix == "" for suffix in filename_pattern_dict.values()):
```

Comment:
If `filename_pattern_dict` modified as suggested, the test `all(...)` seems wrong.

Reply:
yes, you are right @sdesrozis. I can push the updated code if the above suggestions are ok.

Reply:
Feel free to push your modifications 😉 We will review the commit again.

Reply:
Thanks, @sdesrozis. I have updated the PR with latest suggested changes.